lm75bd/lm75bd.c

```diff
@@ -27,7 +27,20 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  uint8_t buf[1];
+  uint8_t buf_1[2];
+  uint8_t tempRegister = 0x00;
+  uint8_t numbytes = 1;
+  buf[0] = tempRegister;
+  i2cSendTo(devAddr, buf, 1);
+  i2cReceiveFrom(devAddr, buf_1, 2);
```

Comment:
- [ ] wrap each line into a separate RETURN_IF_ERROR_CODE macro, we don't want to continue if there is an error
- [ ] use sizeof(buf) instead of 1
- [ ] use sizeof(buf_1) instead of 2